lib/MPD.pm

```diff
@@ -38,15 +38,16 @@ has TwoBitFile  => ( is => 'ro', isa => AbsFile, coerce => 1, required => 1, );
 has MpdBinary   => ( is => 'ro', isa => AbsFile, coerce => 1, required => 1, );
 has MpdIdx      => ( is => 'ro', isa => File,    coerce => 1, required => 1, );
 has dbSnpIdx    => ( is => 'ro', isa => File,    coerce => 1, required => 1, );
-has OutExt => ( is => 'ro', isa => 'Str', required => 1, );
-has OutDir => ( is => 'ro', isa => AbsPath, coerce => 1, required => 1, );
-has Debug    => ( is => 'ro', isa => 'Bool', default => 0 );
-has RunIsPcr => ( is => 'ro', isa => 'Bool', default => 1 );
-has Act      => ( is => 'ro', isa => 'Bool', default => 0 );
+has timeout     => ( is => 'ro', isa => 'Int',   required => 1, );
```

Comment:
In general, maybe easier to review if place new param at end.

I also think we shouldn't require this; lets provide a default value

Reply:
Agreed.  I had started to implement a default within the function itself but putting here is much cleaner.  Should not be required regardless.